src/main/groovy/com/github/jrubygradle/jar/JRubyJarTask.groovy

```diff
@@ -19,4 +19,14 @@ class JRubyJar extends Jar {
     JRubyJar() {
         appendix = 'jruby'
     }
+
+    private JRubyJarConfigurator configurator
+
+    void setConfigurator(JRubyJarConfigurator configurator) {
+        this.configurator = configurator
+    }
+
+    JRubyJarConfigurator getConfigurator() {
```

Comment:
I realise by looking at this now, that I have not had a good look at a previous merged PR. First up, is to say that I would not not do it this way. I assume this was modelled on the previous configurator in this project where an extension was added to the existing `Jar` task. Now that you have created your own task class, that inherits from `Jar`, you have it a lot easier andtThere is no need to have a separate configurator in this case. I would suggest that you simply merge ervything from the `JRubyJarConfigurator` class into this `JRubyJar` class. It will lead to a much simpler DSL for script authors.